test/test_dry.py

```diff
@@ -41,6 +42,8 @@ def enable_all_progression_location_options(options: Options):
 
 def enable_all_options(options: Options):
   enable_all_progression_location_options(options)
+  # options.progression_locations
+  # options.excluded_locations
```

Comment:
Did you mean to leave these comments in?

Reply:
Yeah, I figured it was in the style of the test function to have all the options there, but the unmodified ones commented out. Like with `randomized_gear`/`starting_gear`.